server/database/queries/User/AddUser.js

```diff
@@ -0,0 +1,12 @@
+const connection= require('../../config/connection');
+
+const addUser = (userData, password)=>{
+    const sqlQuery = {
+        text: 'INSERT INTO users (name, email, password) VALUES ($1,$2,$3) RETURNING *',
```

Comment:
If you are not using the inserted values then there's no need to return them

Reply:
done